src/frequenz/sdk/actor/_power_managing/_bounds_methods.py

```diff
@@ -0,0 +1,143 @@
+# License: MIT
+# Copyright © 2023 Frequenz Energy-as-a-Service GmbH
+
+"""Methods for checking and clamping bounds and power values to exclusion bounds."""
+
+from ...timeseries import Bounds, Power
+
+
+def _check_exclusion_bounds_overlap(
```

Comment:
I would make the symbols public now (no underscore), as they are meant to be imported from other modules. Having the module *private* (starting with `_`) should be enough to hide it from the end user interface.

Reply:
done